draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -2019,6 +2023,34 @@ Author/Change controller:
 Specification document:
 : this specification ({{set-cookie}})
 
+## Cookie Attribute Registry
+
+The "Cookie Attribute Registry" defines the set of attribute names used to
+control cookies' behavior. The registry is maintained at <https://example.com/>.
+
+### Procedure
+
+Each registered attribute name is associated with a description, and a
+reference detailing how the attribute is to be processed and stored.
+
+New registrations happen on a "Standards Action" basis (see Section 4.1 of
```

Comment:
So, Standards Action is pretty much the highest bar, and would preclude a document submitted on the Independent Stream (since it requires IETF Stream), and as an Internet Standard.

IETF Review would still require IETF Stream, but would allow e.g., the HTTP WG to publish it as Informational.

RFC Required would allow any RFC from any stream. 

The other option would be Expert Review, but that would mean you'd need to define the review criteria for the expert. 

I'd recommend one of the latter two.

Reply:
Thank you for the clarification. RFC Required sounds reasonable to me. I've switched this to point to that mechanism.